test/validation.test.js

```diff
@@ -1287,6 +1290,39 @@ testProp(
   }
 );
 
+/**
+ * Tests for validName
+ */
+test('validName validator produces correct validator object', (t) => {
+  const invalidNameString = ` `;
+  t.is(validName.error, VALID_NAME_ERROR);
+  t.deepEqual(validName(invalidNameString), {
+    type: VALID_NAME,
+    args: [invalidNameString],
+    error: VALID_NAME_ERROR,
+  });
+});
+
+test('validName validator accepts when value is valid name', (t) => {
+  t.is(validatorFns[VALID_NAME](`James`, ['doesntmatter'], {}), true);
+});
+
+test('validName validator accepts a space between multiple names', (t) => {
+  t.is(validatorFns[VALID_NAME](`Mary Beth`, ['doesntmatter'], {}), true);
+});
+
+test('validName validator accepts an apostrophe', (t) => {
+  t.is(validatorFns[VALID_NAME](`O'Connor`, ['doesntmatter'], {}), true);
+});
+
+test('validName validator rejects an empty space', (t) => {
+  t.is(validatorFns[VALID_NAME](` `), false);
+});
+
+test('validName validator rejects an empty field', (t) => {
+  t.is(validatorFns[VALID_NAME](``), false);
```

Comment:
using a property-based testing technique might be cool here and expose necessary additions to the regex. non-blocking.